docs/MatrixOne/Develop/schema-design/1.1-vector.md

```diff
@@ -0,0 +1,176 @@
+# Vectors
+
+## What Are Vectors
+
```

Comment:
Data content, including text and audio, can be transformed into numerical representations known as vector embeddings using embedding models. These embeddings can be stored in databases and queried to find nearest neighbors, enhancing content search capabilities. Additionally, language models can augment this functionality by generating data based on these embeddings.

Reply:
I tweaked a few words: A vector is a numerical representation encodes data content, such as text or audio, using embedding models. These vectors can be effectively stored and queried alongside relational data. Vector queries encompass tasks like locating nearest neighbors, which can significantly improve search retrievals, as seen in applications like facial recognition or enhancing Generative AI results.